src/styles/variables.scss

```diff
@@ -21,6 +21,9 @@ $color-silver: #cbcbcb;
 $color-light-grey: #babac0;
 $color-medium-grey: #373737;
 $color-dark-grey: #787878;
+$color-darker-grey: #161616;
+$color-darker-grey-hover: #272727;
```

Comment:
I know that we already have the `$color-dark-grey` but I think we should stick to "from general to specific" rule sam as you did with `$color-red-dark`. So in this case:
`$color-grey-darker` and `$color-grey-darker-hover`.
And the `$color-dark-grey` could be refactored then.
What do you think?